src/scss/index.scss

```diff
@@ -207,7 +211,7 @@ td:first-child {
 
 .navbar-light {
   background: #fff !important;
-  box-shadow: 0 0.125rem 0.25rem 0 rgba(0, 0, 0, 0.11);
+  box-shadow: 0 0.125rem 0.25rem -0.125rem rgba(0, 0, 0, 0.11);
```

Comment:
this makes the drop-shadow on the navbar point straight down

Reply:
This makes the shadow less prominent, that is on purpose? (no strong opinion)

Reply:
It's not the intention, it was actually only necessary due to the centering (so that the top-bar didn't have shadows to the right and left, as they do now)

Reply:
How about I just strip away everything except for removing the `max-width` of `.bd-content`? I feel like that is an important bugfix that's getting bogged down by these other design questions

Reply:
> How about I just strip away everything except for removing the max-width of .bd-content? I feel like that is an important bugfix that's getting bogged down by these other design questions

Yep, +1